tests/data/instances/sequence_tagging/verb_semantics_instance_test.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=no-self-use,invalid-name
+from typing import List
+
+from numpy.testing import assert_array_almost_equal
+
+from deep_qa.data.data_indexer import DataIndexer
+from deep_qa.data.instances.instance import TextInstance
+from deep_qa.data.instances.sequence_tagging.pretokenized_tagging_instance import PreTokenizedTaggingInstance
+from deep_qa.data.instances.sequence_tagging.verb_semantics_instance import VerbSemanticsInstance
+from deep_qa.data.tokenizers import tokenizers
+from deep_qa.common.params import Params
+from ....common.test_case import DeepQaTestCase
+
+
+class TestVerbSemanticsInstance(DeepQaTestCase):
```

Comment:
You might want to test that when you call `to_indexed_instance`, everything functions as you expect - you can have a look at an example of that here: 
https://github.com/allenai/deep_qa/blob/master/tests/data/instances/sequence_tagging/test_tagging_instance.py#L24